packages/analytics/src/web.ts

```diff
@@ -106,4 +110,21 @@ export class FirebaseAnalyticsWeb
   public async resetAnalyticsData(): Promise<void> {
     throw this.unimplemented('Not implemented on web.');
   }
+
+  public async initiateOnDeviceConversionMeasurementWithEmail(options: InitiateOnDeviceConversionMeasurementWithEmailOptions): Promise<void> {
+    throw this.unimplemented('Not implemented on web.' + JSON.stringify(options));
```

Comment:
```suggestion
    throw this.unimplemented('Not implemented on web.');
```

Please just return the message without options.